src/FlatSharp.Runtime/IO/SpanWriterExtensions.cs

```diff
@@ -36,6 +38,27 @@ public static class SpanWriterExtensions
 
         memory.Span.CopyTo(span.Slice(vectorStartOffset + sizeof(uint)));
     }
+    
+    public static void UnsafeWriteSpan<TSpanWriter, TElement>(
```

Comment:
You actually don't need to pass all of these fields if you don't want. This is just the "default" serialization signature for FlatSharp's generated methods, but since this in effect a leaf node, feel free to throw other items in this signature as you need, such as item alignment, etc.